datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -733,6 +733,10 @@ def process_metric(self, metric, scraper_config, metric_transformers=None):
                     self.log.warning('Error handling metric: %s - error: %s', metric.name, err)
 
                 return
+            # check for wilcards in transformers
+            for transformer_name, transformer in iteritems(metric_transformers):
+                if transformer_name.endswith('*') and metric.name.startswith(transformer_name[:-1]):
+                    transformer(metric, scraper_config, transformer_name)
```

Comment:
did you branch off the old branch? you can rebase with master and remove any conflicts
```
git rebase master
```

Reply:
Ok I see what I did here, I think I fixed this now.